src/app/shared/task-provider/task-provider.component.html

```diff
@@ -39,13 +39,36 @@
                 </div>
             </div>
         </div>
+        <mat-toolbar class="small-toolbar">
+            Tracker Ticket Status Mapping
+        </mat-toolbar>
+        <div class="table-box padded-container" [formGroupName]="taskProviderConfigKey">
+            <div class="layout-row">
+                <mat-form-field class="flex">
+                    <input matInput 
+                    placeholder="Done (Comma Separated)" 
+                    [pattern]="commaSeparatedRegex" 
```

Comment:
Change the regex to handle the following special characters `'`, `-`, `_`.  For example, the regex breaks for `Won't Fix` status for JIRA.